ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMTQwIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9lLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhbm9uSWQiOiI1ZTg3MTYwMDcxZjY3N2M4In0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6M30sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyMTQwIiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyMTQwIl19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMjc0OTQ3MzcyODk2OTQyNiJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5592c1bafcea <unknown>
#1 0x5592c16605f0 <unknown>
#2 0x5592c1646e86 <unknown>
#3 0x5592c164579a <unknown>
#4 0x5592c1645eff <unknown>
#5 0x5592c1645e54 <unknown>
#6 0x5592c16647ff <unknown>
#7 0x5592c17099aa <unknown>
#8 0x5592c16d7662 <unknown>
#9 0x5592c16fd660 <unknown>
#10 0x5592c16d7433 <unknown>
#11 0x5592c16a3ea3 <unknown>
#12 0x5592c16a4b01 <unknown>
#13 0x5592c1b74b3b <unknown>
#14 0x5592c1b78a21 <unknown>
#15 0x5592c1b5bc32 <unknown>
#16 0x5592c1b79594 <unknown>
#17 0x5592c1b3feef <unknown>
#18 0x5592c1b9dd98 <unknown>
#19 0x5592c1b9df76 <unknown>
#20 0x5592c1baeb36 <unknown>
#21 0x7fd6a629caa4 <unknown>
#22 0x7fd6a6329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zYWxvb24vZS1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJ1c2VyIjp7ImFub25JZCI6IjVlODcxNjAwNzFmNjc3YzgifSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjozfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI5NTAyMzMxMTQxMjg5MjUxIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5592c1bafcea <unknown>
#1 0x5592c16605f0 <unknown>
#2 0x5592c1646e86 <unknown>
#3 0x5592c164579a <unknown>
#4 0x5592c1645eff <unknown>
#5 0x5592c1645e54 <unknown>
#6 0x5592c16647ff <unknown>
#7 0x5592c17099aa <unknown>
#8 0x5592c16d7662 <unknown>
#9 0x5592c16fd660 <unknown>
#10 0x5592c16d7433 <unknown>
#11 0x5592c16a3ea3 <unknown>
#12 0x5592c16a4b01 <unknown>
#13 0x5592c1b74b3b <unknown>
#14 0x5592c1b78a21 <unknown>
#15 0x5592c1b5bc32 <unknown>
#16 0x5592c1b79594 <unknown>
#17 0x5592c1b3feef <unknown>
#18 0x5592c1b9dd98 <unknown>
#19 0x5592c1b9df76 <unknown>
#20 0x5592c1baeb36 <unknown>
#21 0x7fd6a629caa4 <unknown>
#22 0x7fd6a6329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9lLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sInVzZXIiOnsiYW5vbklkIjoiNWU4NzE2MDA3MWY2NzdjOCJ9LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjY4NDU5Njc0MTg0MjAwMjgiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5592c1bafcea <unknown>
#1 0x5592c16605f0 <unknown>
#2 0x5592c1646e86 <unknown>
#3 0x5592c164579a <unknown>
#4 0x5592c1645eff <unknown>
#5 0x5592c1645e54 <unknown>
#6 0x5592c16647ff <unknown>
#7 0x5592c17099aa <unknown>
#8 0x5592c16d7662 <unknown>
#9 0x5592c16fd660 <unknown>
#10 0x5592c16d7433 <unknown>
#11 0x5592c16a3ea3 <unknown>
#12 0x5592c16a4b01 <unknown>
#13 0x5592c1b74b3b <unknown>
#14 0x5592c1b78a21 <unknown>
#15 0x5592c1b5bc32 <unknown>
#16 0x5592c1b79594 <unknown>
#17 0x5592c1b3feef <unknown>
#18 0x5592c1b9dd98 <unknown>
#19 0x5592c1b9df76 <unknown>
#20 0x5592c1baeb36 <unknown>
#21 0x7fd6a629caa4 <unknown>
#22 0x7fd6a6329c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9lLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6IncyMTQiLCJ2ZWhpY2xlQk0iOiIyMTQwIiwidmVoaWNsZVBJSGlzdG9yeSI6IjIxNDAiLCJ0aW1lc3RhbXAiOjE3NDU0NDA3ODc0NjUsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxOTc3OTI2MzIuMTc0NTQ0MDc4NSJ9LCJhbm9uSWQiOiI1ZTg3MTYwMDcxZjY3N2M4In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3NhbG9vbi9lLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI4NDY0Nzg5MTA0OTE4NzQ0IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x5592c1bafcea <unknown>
#1 0x5592c16605f0 <unknown>
#2 0x5592c1646e86 <unknown>
#3 0x5592c164579a <unknown>
#4 0x5592c1645eff <unknown>
#5 0x5592c1645e54 <unknown>
#6 0x5592c16647ff <unknown>
#7 0x5592c17099aa <unknown>
#8 0x5592c16d7662 <unknown>
#9 0x5592c16fd660 <unknown>
#10 0x5592c16d7433 <unknown>
#11 0x5592c16a3ea3 <unknown>
#12 0x5592c16a4b01 <unknown>
#13 0x5592c1b74b3b <unknown>
#14 0x5592c1b78a21 <unknown>
#15 0x5592c1b5bc32 <unknown>
#16 0x5592c1b79594 <unknown>
#17 0x5592c1b3feef <unknown>
#18 0x5592c1b9dd98 <unknown>
#19 0x5592c1b9df76 <unknown>
#20 0x5592c1baeb36 <unknown>
#21 0x7fd6a629caa4 <unknown>
#22 0x7fd6a6329c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/BFV1_attempt_1.png'